.changeset/config.json

```diff
@@ -17,5 +17,6 @@
   "access": "public",
   "baseBranch": "dev",
   "updateInternalDependencies": "patch",
+  "bumpVersionsWithWorkspaceProtocolOnly": true,
```

Comment:
This setting is now enabled so that Changesets doesn't attempt to bump the `react-router-dom` dependency in `react-router-dom-v5-compat` since it's set to `"4 || 5"`. This is a simpler method of achieving what the `postversion.mjs` script was doing since we can now use `workspace` dependencies.